### PR TITLE
Merge Rook upstream to Koor

### PR DESCRIPTION
**Pull request by galexrt**
Created by Github action Backport rook/rook upstream cronjob

### CODE REVIEW

**Review by zalsader — APPROVED** [review]
I'm wondering now if sso is still working after this. Do we have a test for it?

File: Documentation/Getting-Started/glossary.md [hunk 1]
```diff
@@ -98,12 +98,6 @@ A stretched cluster is a deployment model in which two datacenters with low late
 
 The [Rook toolbox](../Troubleshooting/ceph-toolbox.md) is a container with common tools used for rook debugging and testing.
 
-## Koor
-
-### Koor Storage Distribution (KSD)
-
-Koor Storage Distribution (KSD) is a product based on Rook Ceph that makes running and maintaining Ceph storage on Kubernetes easier.
-
 ## Ceph
```
[the remark this answers is not in the document]
Comment (galexrt) [reply Documentation/Getting-Started/glossary.md:101]: I think I found a fix for that, will tweak the rebase script in the merge commit

**Comment by galexrt** [comment]
@zalsader PTAL

**Review by zalsader — APPROVED** [review]

**Comment by zalsader** [comment]
What's up with License Compliance?

**Comment by galexrt** [comment]
I have checked on their webinterface and the dependency is just a "flagged" one that is no concern for us.

**Pull request merged**